README.md

```diff
@@ -14,10 +14,11 @@ The goal of Parsec is to:
 * Minimize time and effort spent on common repetitive tasks
 * Provide helper [libraries and utilities](https://github.com/yahoo/parsec-libraries) for common tasks
 * Reduce the learning curve and maintenance cost
+* Provide client for your web service
```

Comment:
could it be modified to be more specific or clear?
e.g. Provide autogenerated Java client for your web service?